tools/SourceKit/include/SourceKit/Core/Context.h

```diff
@@ -24,14 +25,17 @@ namespace llvm {
 }
 
 namespace SourceKit {
-  class LangSupport;
-  class NotificationCenter;
+class FileSystemProvider;
+class LangSupport;
+class NotificationCenter;
```

Comment:
I'm not sure about the formatting conventions in this repo, so I won't be commenting on that aspect of the PR. I guess it may be desirable to keep original formatting but manually undoing changes done by clang-format is probably not a very pleasant experience.

Reply:
I'm going to undo them all